build.gradle

```diff
@@ -82,7 +82,7 @@ configurations {
 }
 
 allprojects {
-    if (!(project.name.endsWith("-dependencies") || project.name.endsWith("-dependency-versions"))) {
```

Comment:
Why don't we exclude the -dependencies modules any more?

Reply:
Because by doing that, we disable the dependency management of the Spring Boot plugin, hence soncarqube can't find the spring dependencies. The other workaround was to add the `valtimo-dependency-versions` as a dependency to `valtimo-(gzac-)dependencies`, but that would turn them into a BOM/POM yet again.